### ISSUE TITLE
Can't update MA from HACs

### ISSUE THREAD

**Issue by SeByDocKy** [music-assistant/hass-music-assistant#880]
### What version of Music Assistant has the issue?

2022.7.2

### The problem

After analysing logs in HACs... it seems that I am trying without success to update MA with patch-2022.7.3. Seems this file is no more present in the github. Is possible to have it availabe in otder to have a normal MA update ? 

### How to reproduce

Just try to click on the update button

### Relevant log output

022-08-24 15:53:34.883 ERROR (MainThread) [custom_components.hacs] <Integration music-assistant/hass-music-assistant> GitHub returned 404 for https://api.github.com/repos/music-assistant/hass-music-assistant/git/trees/patch-2022.7.3

 "pushed_at": "2022-08-23T21:18:45",
      "releases": true,
      "selected_tag": "patch-2022.7.3",
      "version_installed": "2022.7.2",
      "last_fetched": 1661424301.722832

### What version of Home Assistant Core are your running

20228.6

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

ODROID

**Comment by OzGav** [comment]
You need to update MA to 2022.8.x as you are running HA 2022.8.6. That option should be in HACS?

**Comment by SeByDocKy** [comment]
No I can't update anything .... that's the main problem since more than in month.... 
When I click on update I got this error file in hacs pointing the missing file....

**Comment by marcelveldt** [comment]
No, we can't bring that file back. Besides that it is an old version too.
Can't you just remove Music Assistant completely from HA and HACS and reinstall ?
Or press the button "Update information" first ?

**Comment by SeByDocKy** [comment]
> No, we can't bring that file back. Besides that it is an old version too. Can't you just remove Music Assistant completely from HA and HACS and reinstall ? Or press the button "Update information" first ?

I tried all your options.... nothing worked unfortunatly .... :(

**Comment by marcelveldt** [comment]
Are you on HA version 2022.8.x ?

**Comment by SeByDocKy** [comment]
Yes everything is updated from my side, last HA, last everything......
 I don't know in the ./storage/hacs.repositories   mentioned with patch file.... impossible to update it manually for another version...

**Comment by marcelveldt** [comment]
Maybe try to update it manually ?
Download the zipfile for the latest release: https://github.com/music-assistant/hass-music-assistant/releases/download/2022.8.4/mass.zip

unpack it in the custom_components folder, overwriting the existing content

**Comment by SeByDocKy** [comment]
I already tried that option too :( :(

**Comment by marcelveldt** [comment]
In that case the only option is to remove HACS completely and reinstall. Also delete all the HACS related files and folders.
We can't help you on this I'm afraid as it is strictly taken an HACS issue and not a MA issue.

**Comment by SeByDocKy** [comment]
I see ... in Hacs issue, they said it's a MA issue (deleted file).... If I delted all ./storage file, I will also loose all other integrations inputs  and reinstalled them too...

**Comment by marcelveldt** [comment]
It is not too uncommon that a release is replaced. Bad things happen and code needs to be guarded for that. I find it kind of strange that the whole thing is unrecoverable crashed due to the fact that we deleted a release 10 minutes after it was published (because it was faulty). I really do not understand why HACS keeps grabbing a tag that does not exists, in my opinion that is a bug in HACS.

This is why we have backups so we can restore after something bad happened ;-)

I have recreated that missing tag but I don't think that will fix the issue.
